Exercise_2.py

```diff
@@ -1,17 +1,79 @@
 
+'''
+// Time Complexity :
+    push    : O(1)
+    pop     :  O(N) N - Number of elements in LinkedList
```

Comment:
Please check the `push`  Time complexity. Also try to implement `O(1)` for both push and pop operations for a stack.

Reply:
Reimplemented in O(1), Thanks for mentioning